packages/cli-template-contracts-foundry/remappings.txt

```diff
@@ -0,0 +1,4 @@
+@semaphore/contracts/=../../node_modules/@semaphore-protocol/contracts/
+@zk-kit/lean-imt.sol/=../../node_modules/@zk-kit/lean-imt.sol/
+forge-std/=../../node_modules/forge-std/src/
+poseidon-solidity/=../../node_modules/poseidon-solidity/
```

Comment:
```suggestion
@semaphore/contracts/=./node_modules/@semaphore-protocol/contracts/
@zk-kit/lean-imt.sol/=./node_modules/@zk-kit/lean-imt.sol/
forge-std/=./node_modules/forge-std/src/
poseidon-solidity/=./node_modules/poseidon-solidity/
```

Are those remappings working for you? These are the remappings working for me.

Reply:
updated